clients/src/main/java/org/apache/kafka/clients/admin/AdminClient.java

```diff
@@ -87,6 +87,9 @@ public void close() {
     /**
      * Create a batch of new topics with the default options.
      *
+     * Similar to #{@link AdminClient#createTopics(Collection, CreateTopicsOptions)},
+     * but uses the default options. See {@link #createTopics(Collection, CreateTopicsOptions)} for more details.
```

Comment:
Actually, took a closer look and it's a bit repetitive to link to the method twice. We should just do it once. Maybe something like:

```java
* This is a convenience method for #{@link AdminClient#createTopics(Collection, CreateTopicsOptions)} with default options. See the overload for more details.
```

Or something like that. What do you think?